### PR TITLE
Add X-Contentful-User-Agent Header

### PR DESCRIPTION
**Pull request by dlitvakb**

### CODE REVIEW

File: contentful/client.py [hunk 1]
```diff
@@ -370,14 +383,54 @@ def _cache_content_types(self):
...
+            header['integration'] = {
+                'name': self.integration_name,
+                'version': self.integration_version
+            }
+        header['platform'] = {
+            'name': 'python',
+            'version': platform.python_version()
+        }
+        header['os'] = {
+            'name': platform.system(),
```
Comment (realityking) [inline contentful/client.py:409]: According to the docs for `platform.system()`

> An empty string is returned if the value cannot be determined.

Should that be handled?

File: contentful/client.py [hunk 2]
```diff
@@ -370,14 +383,54 @@ def _cache_content_types(self):
...
+
     def _request_headers(self):
         """
         Sets the default Request Headers.
         """
 
-        from . import __version__
         headers = {
-            'User-Agent': 'PythonContentfulClient/{0}'.format(__version__),
+            'X-Contentful-User-Agent': self._contentful_user_agent(),
```
Comment (realityking) [inline contentful/client.py:433]: It's just my OCD talking but I think you might have a trailing space here, I'd trim it but that's really just me :D

**Review by realityking — COMMENTED** [review]
Otherwise looks good 👍

File: contentful/client.py (hunk 2 above)
Comment (dlitvakb) [reply contentful/client.py:433]: Hey @realityking, there's no trailing space.

File: contentful/client.py (hunk 1 above)
Comment (dlitvakb) [reply contentful/client.py:409]: Done, handled below when appending the sections.

Comment (realityking) [reply contentful/client.py:409]: 👍

**Review by realityking — APPROVED** [review]
LGTM

**Pull request merged**